package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.8.7",
   "description": "Reliable money math with BigNumber  wrapped inside Cents",
   "author": "Good Eggs <open-source@goodeggs.com>",
+  "type": "module",
```

Comment:
Is this the only change required to start using `import` instead of `require` ?

Reply:
You don't need any change like this. It should just work with the babel config file you wrote.

I think you're getting confused because you're also trying to use ES modules in `.eslintrc.js` itself. Is that right? `.eslintrc` files don't support ES modules – that's why we have [this override to allow CommonJS in those files](https://github.com/goodeggs/goodeggs-money/blob/9f2c3043ff9c47ea841e98f0dea3c276c85d8072/.eslintrc.js#L15-L24)

Reply:
🤔 that is what I though initially, but without adding this `"type": "module"` I was getting 
```error
import {BigNumber} from 'bignumber.js';
^^^^^^

SyntaxError: Cannot use import statement outside a module
```
so I added this thinking this must converted to ES Module.

That being said, or I'm missing something else or somehow the override is not working.

Reply:
what's throwing this? eslint? mocha? babel build step?

Reply:
come to think of it, if the error is with mocha, that might be the problem... we forget to tell mocha to use babel:
https://github.com/goodeggs/crashpad/blob/8fc1fd8eac650d523e1d5651fdf4cfaa569c5dc9/package.json#L37

Reply:
Yes, looks like to be mocha, from your sample I added 
`mocha --require @babel/polyfill --extension ts,js,jsx,tsx`
but still getting the same, I just removed the `--require` for register file, since we don't need that yet, right?

<img width="1050" alt="image" src="https://user-images.githubusercontent.com/5542631/198133494-f171fbc1-dfe3-4c5d-b7eb-52cc7969a992.png">

Reply:
no, the important part is the babel register: `--require .register.ts` and create a file like this: https://github.com/goodeggs/crashpad/blob/8fc1fd8eac650d523e1d5651fdf4cfaa569c5dc9/.register.ts. You can put `['.ts', '.js']` for now and remove `.js` in the next PR.

This is what hooks up babel code transpilation into mocha.

Reply:
That's it, thanks a lot 🙌